doc/source/tutorial/fft.rst

```diff
@@ -431,7 +431,7 @@ provides a five-fold compression rate.
     >>> from scipy.fft import dct, idct
     >>> import matplotlib.pyplot as plt
     >>> N = 100
-    >>> t = np.linspace(0,20,N)
+    >>> t = np.linspace(0,20,N, endpoint=False)
```

Comment:
This changes the doctest values on lines 441 and 448. You should just be a be able to copy the values from [CI](https://travis-ci.org/github/scipy/scipy/jobs/730091457#L1572) or run the code yourself.